README.md

```diff
@@ -17,7 +17,7 @@ Bundler helps manage dependencies for Ruby projects. Find more info here: <http:
 Add this gem to your `Gemfile`:
 
 ```ruby
-gem "opentok", "~> 2.4"
+gem "opentok", "~> 2.4.1"
```

Comment:
@Lucashuang0802 I don't agree with >= because when we release a major version it can break people's apps unexpectedly. ~> gives us the power of semver to prevent this unexpected breakages.